src/ObjectLock.php

```diff
@@ -247,7 +247,7 @@ function askUnlock() {
                      cache: false,
                      data: {
                         requestunlock: 1,
-                        id: {$this->fields['id']}
+                        id: {$this->itemid}
```

Comment:
The `requestunlock` action will try to load the `ObjectLock` instance using `(new ObjectLock())->getFromDB($_POST["id"])`, so, unless I miss something, the you have to pass `$this->fields['id']` to the action.

Reply:
When you use id: {$this->fields['id']} the sended id is 1 and not the ticket's id

Reply:
You're right :)
Better fix :P

Reply:
You have this error if you not do this : 

```
[2023-11-10 07:27:20] glpiphplog.WARNING:   *** PHP Warning (2): Undefined array key "date_mod" in C:\wamp64\www\glpi10\src\NotificationTargetObjectLock.php at line 122
  Backtrace :
  src\NotificationTarget.php:1363                    NotificationTargetObjectLock->addDataForTemplate()
  src\NotificationTemplate.php:278                   NotificationTarget->getForTemplate()
  src\NotificationEventAbstract.php:122              NotificationTemplate->getTemplateByLanguage()
  src\NotificationEvent.php:187                      NotificationEventAbstract::raise()
  ajax\unlockobject.php:78                           NotificationEvent::raiseEvent()
```